atom_batch_execution/scripts/plot_graphs

```diff
@@ -0,0 +1,325 @@
+
+
+
+def computeLambdaConditions(line_dict,args):
+    '''
+    This function will be used to filter which files to take data from
+    '''
+
+    def addConditionToLambda(lambda_expression,condition,operator = "and"):
+
+        if lambda_expression == "":
+            lambda_expression = f"lambda x : {condition}"
+        else:
+            lambda_expression += f" {operator} {condition}"
+
+        return lambda_expression
+
+
+    lambda_expression = ""
+
+    for key,value in line_dict.items():
+
+        # These two keys determine which field to plot, it's not relevant here
+        if key == "ydata" or key == "xfield" or key == "legend":
+            continue
+
+        # Lists determine a range of values to plot
+        if isinstance(value,list):
+            lambda_expression = addConditionToLambda(lambda_expression,f"x['{key}'] >= {value[0]}","and")
+            lambda_expression = addConditionToLambda(lambda_expression,f"x['{key}'] <= {value[1]}","and")
+
+        elif type(value) in (int,float):
+            lambda_expression = addConditionToLambda(lambda_expression,f"x['{key}'] == {value}","and")
+
+        # A bool determines whether to include all available values of a field or not
+        # Being false is equal to not being declared
+        # https://stackoverflow.com/questions/37888620/comparing-boolean-and-int-using-isinstance
+        elif type(value) == bool:
+            # Retrieve the value, if the key doesn't exist .get() returns None
+            if value : 
+                lambda_expression = addConditionToLambda(lambda_expression,f"x.get('{key}')","and")
+            else:
+                lambda_expression = addConditionToLambda(lambda_expression,f"not x.get('{key}')","and")
+
+        # Any other key that isn't listed as a bool or list should be added to the blackblist as well
+        else:
+                lambda_expression = addConditionToLambda(lambda_expression,f"not x.get('{key}')","and")
+
+    if args["verbose"]:
+        print(f"Printing lambda used in filtering files {Fore.BLUE}{lambda_expression}{Style.RESET_ALL}")
+
+
+    lambda_function = eval(lambda_expression)
+
+    return lambda_function,lambda_expression
+
+
+
+
+def main():
+
+    ap = argparse.ArgumentParser()  # Parse command line arguments
+    ap.add_argument("-v", "--verbose", help="Prints the stdout_data of each command to the terminal.",
+                    action='store_true', default=False)
+    ap.add_argument("-ow", "--overwrite", help="Overwrites output folder.",
+                    action='store_true', default=False)
+    ap.add_argument("-rf", "--results_folder", help="Folder containing the processed results",
+                    required=True, type=str)
+    ap.add_argument("-of", "--output_folder", help="Folder where to store the plots' images",
+                    required=True, type=str)
+    ap.add_argument("-df", "--data_filename", help="Yaml containing variables used to compute the plots.",
```

Comment:
This should be documented and standardized.

Reply:
https://github.com/lardemua/atom/blob/fc1ea8fa0aad4b0495d968753a43f594d1604064/atom_batch_execution/scripts/batch_execution#L65-L68

I'm pretty sure its following the same structure as in the `batch_execution` script.